returnn/tf/layers/basic.py

```diff
@@ -4499,19 +4499,32 @@ def find_axis(a_axis, b_axis):
     if not b_var_dims and add_var2_if_empty:
       b_var_dims.append(1)
 
+    def get_batch_axis_excluding_batch(axis):
+      """
+      :param int axis:
+      :return int:
+      """
+      if batch_dim_axis is None:
+        return axis
+      if axis == batch_dim_axis:
+        return None
```

Comment:
This is a case which should not happen here, right? (Also the docstring would be wrong then. If this is allowed, the return type would be `int|None`.)
```suggestion
      assert axis != batch_dim_axis
```